br/pkg/backup/store.go

```diff
@@ -139,7 +137,11 @@ func doSendBackup(
 		}
 		time.Sleep(3 * time.Second)
 	})
+	reqStartKey, reqEndKey := req.StartKey, req.EndKey
 	bCli, err := client.Backup(ctx, &req)
+	// Note: derefer req here to let req.SubRanges be released as soon as possible.
```

Comment:
However, it is not released quickly because the variable `req` is also held by retry logic. Maybe we need to remove the retry logic. See https://github.com/pingcap/tidb/issues/60309